api/apps/api/src/modules/users/users.controller.ts

```diff
@@ -125,4 +131,55 @@ export class UsersController {
   ): Promise<void> {
     return this.service.markAsDeleted(req.user.id);
   }
+
+  @ApiOperation({
+    description: 'Get list of admins in platform.',
+  })
+  @ApiOkResponse()
+  @ApiUnauthorizedResponse()
+  @ApiForbiddenResponse()
+  @Get('admins')
+  async getAdmins(
+    @Request() req: RequestWithAuthenticatedUser,
+  ): Promise<PlatformAdminEntity[]> {
+    const result = await this.service.getPlatformAdmins(req.user.id);
+    if (isLeft(result)) {
+      throw new ForbiddenException();
+    }
+    return result.right;
+  }
+
+  @ApiOperation({
+    description: 'Add user as admin.',
```

Comment:
maybe (mirroring the existing operation description in the delete handler below
```suggestion
    description: 'Grant platform admin role to user.',
```